.github/CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,5 @@
 # Community Code of Conduct
 
-Please see the official [Ansible Community Code of Conduct][CoC].
```

Comment:
This doesn't make sense to lowercase — it's a commonly used abbreviation.

Reply:
That is not user visible. Prettier seems to enforce use of lowercase reference keys and is not configurable. I do not see any problem with that. I do not this that either as actionable or fixable.

Reply:
It's a big problem because it adds friction to cross repo sync + breaks expectations of identifiers being searchable with a standard case-sensitive search.